ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -267,6 +267,19 @@ void OwInterface::initialize()
 {
   LanderInterface::initialize();
 
+  std::string fault_hierarchy_path;
+  bool verbose_flag = false;
+  if(m_genericNodeHandle->getParam("/verbose_fault_hierarchy_flag", verbose_flag)){
+    verbose_flag = verbose_flag;
+  }
```

Comment:
`getParam` sets your passed value by reference, and the return value is false if the parameter could not be found. 
```suggestion
  bool verbose_flag;
  if(!m_genericNodeHandle->getParam("/verbose_fault_hierarchy_flag", verbose_flag)){
    // ROS_ERROR MESSAGE HERE
  }
```